pkg/api/validate/dynamic/dynamic.go

```diff
@@ -55,7 +58,9 @@ type Dynamic interface {
 
 type dynamic struct {
 	log            *logrus.Entry
+	authorizer     refreshable.Authorizer
 	authorizerType AuthorizerType
+	cred           azcore.TokenCredential
```

Comment:
is it fp cred or cluster sp cred? please use an easily identifiable name

Reply:
It could be both like the above `authorizer`.  The caller (dynamic() in `pkg/api/validate/openshiftcluster_validatedynamic.go` specifies which one to use.  It's not fixed to fp or sp.

Reply:
perhaps call it `checkAccessSubjectCred` and add some comments that it may be either cluster SP or FP and is used in check access subject info

Reply:
that makes sense.  done.